third_party/libunwindstack/PeCoffEpilog.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef _LIBUNWINDSTACK_PE_COFF_EPILOG_H
+#define _LIBUNWINDSTACK_PE_COFF_EPILOG_H
+
+#include <vector>
+
+#include <capstone/capstone.h>
+
+#include <unwindstack/Error.h>
+#include <unwindstack/Memory.h>
+#include <unwindstack/Regs.h>
+
+namespace unwindstack {
+
+class PeCoffEpilog {
+ public:
+  ~PeCoffEpilog();
+
+  // Needs to be called before one can use DetectAndHandleEpilog.
+  bool Init();
```

Comment:
Because it can fail, so we'd need some other way to check if initialization was truly successful. It was suggested (I think by @danielfenner) that it would be better to have factory methods instead of the Init() to handle initialization failures, and I agree, but this is the standard way it's done in libunwindstack and I want to keep it this way for consistency.

Reply:
Makes sense. Thanks!